_layouts/page.html

```diff
@@ -59,6 +59,7 @@ <h2>{{ item.title }}</h2>
 </head>
 <body>
   <div id="page-container">
+    <div class="not-supported-banner"> {% unless site.version_list contains page.version %} <center> This version is <a href='/{{site.latest_version}}/support.html'>no longer supported</a>. Click <a href="/{{site.latest_version}}/">here</a> for the latest version. </center>  {% endunless %} </div>  
```

Comment:
```suggestion
     {% unless site.version_list contains page.version %} <div class="not-supported-banner"> <center> This version is <a href='/{{site.latest_version}}/support.html'>no longer supported</a>. Click <a href="/{{site.latest_version}}/">here</a> for the latest version. </center> </div> {% endunless %}   
```

Reply:
Suggest putting the div inside the if statement so that the supported versions don't have an unneeded empty div in the dom.